phpunit

```diff
@@ -11,7 +11,7 @@
  */
 
 // Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2016-03-16 15:36 UTC
+// Cache-Id-Version: 2016-03-16 16:33 UTC
```

Comment:
it's required when you need to force a reload of phpunit or any of its dependencies, not the case here